documentation/GuideLines.md

```diff
@@ -0,0 +1,165 @@
+# DRAFT
+This is a draft of the proposed code standards and best practices for the Mission Control Project going forward to help facilitate a productive cross-team workflow to a unified standard. The plan is to use this standard to outline the workflow milestones for the more long term restructuring of some components needed to fulfill the vision of Mission Control envisioned by the stakeholder.
+
+# GitFlow - Git pull requests practices and standards
+### Create a branch
+* Pull down from Master to your local machine. This will assure that you have the latest and greatest from the repo!
+```
+git pull master
+```
+* Create/checkout a local feature branch. Once you have the master locally, create a branch of your feature.
+```
+git checkout -b feature/"YOUR-FEATURE-NAME"
+```
+
+### Work Flow
+* Be mindful of GitHub updates and announcements posted to the Slack channel.
+* Strive to keep individual contributions under 500 changed lines. 
+* Update Draft Pull Request description to include a detailed description of the new feature or bug fix that being is added.
```

Comment:
It would be good to link to a detailed description.

Reply:
Based on this feedback have added a link to [244](https://github.com/Lambda-School-Labs/mission-control-fe/pull/244).